proof/invariant-abstract/AARCH64/ArchSchedule_AI.thy

```diff
@@ -28,6 +28,7 @@ lemma dmo_mapM_storeWord_0_invs[wp,Schedule_AI_assms]:
    apply wp
   by (simp add: upto.simps word_bits_def)
 
+(* FIXME arch_split: why not arch global naming? this doesn't make sense *)
 global_naming Arch
```

Comment:
agreed, doesn't make sense to me either. Do we access any of these under `Arch.` later?

Reply:
Agreed, very weird

Reply:
fixed